### ISSUE TITLE
Implement mint cap of KUSD

### ISSUE THREAD

**Issue by udosson** [kumodao/kumo-protocol#318]
The ecosystem KUMO is in is very new and lacks deep liquidity on DEXs. The limited liquidity needs to be reflected in minting caps of KUSD per asset, so we prevent economic attacks towards the KUMO protocol, maintain exist-liquidity, and grow together at the speed of the ReFi ecosystem.

Definition of Done:
- Imple

**Comment by marceljay** [comment]
@udosson @arthurka-o Is this still a desired feature? If so, we should track it in Notion.

**Comment by udosson** [comment]
Yes. @arthurka-o - did you implement something related to the mint cap? I think I remember something
